src/components/pages/gallery/Upload.tsx

```diff
@@ -121,11 +147,39 @@ export default function Upload(props: Props) {
         setProgressView(true);
     };
 
+    const uploadFailedFiles = async () => {
```

Comment:
I don't think we need this, What I was thinking was that we could just trigger a new upload by setting electronFiles State with the failed files na ?

Reply:
I don't think this is ideal, because wouldn't the user then will see the whole upload flow again?

Reply:
we can have a special state as we have for the first upload, where we check if it's the first upload then we don't show the collection creation and `collectionSelector` modal and directly create and upload to new collection name "First Album", we can have something like that for this too na?

Reply:
Cool, will do that way.